app/models/Experiments.php

```diff
@@ -182,6 +179,48 @@ public function updateStatus($status)
     }
 
     /**
+    * Returns if this experiment can be timestamped
+    *
+    * @return bool
+    */
+    public function isTimestampable()
+    {
+        $current_status = $this->getCurrentStatus();
+        $sql = "SELECT allow_timestamp FROM status WHERE id = :status LIMIT 1;";
+        $req = $this->pdo->prepare($sql);
+        $req->bindParam(':status', $current_status);
+        $req->execute();
+        $allowTimestamp = $req->fetchColumn();
+        if ($allowTimestamp === '1') {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    /**
+     * Get current experiment status ID
+     *
+     * @return int The status ID
+     */
+    private function getCurrentStatus()
```

Comment:
Yep. Great, I guessed that it's probably already accessible somewhere, but didn't find it either in the docs nor in the code ;)